pull.py

```diff
@@ -83,16 +117,25 @@
     'CC-BY-ND-2.5': {'spdx': 'CC-BY-ND-2.5'},
     'CC-BY-ND-3.0': {'spdx': 'CC-BY-ND-3.0'},
     'CC-BY-ND-4.0': {'spdx': 'CC-BY-ND-4.0'},
+    'CC-BY-SA-1.0': {'spdx': 'CC-BY-SA-1.0'},
+    'CC-BY-SA-2.0': {'spdx': 'CC-BY-SA-2.0'},
+    'CC-BY-SA-2.5': {'spdx': 'CC-BY-SA-2.5'},
+    'CC-BY-SA-3.0': {'spdx': 'CC-BY-SA-3.0'},
+    'CC-BY-SA-4.0': {'spdx': 'CC-BY-SA-4.0'},
     'CC0': {'spdx': 'CC0-1.0'},
     'CDDL': {'spdx': 'CDDL-1.0'},
     'CPAL': {'spdx': 'CPAL-1.0'},
     'CeCILL': {'spdx': 'CECILL-2.0'},
     'CeCILL-B': {'spdx': 'CECILL-B'},
     'CeCILL-C': {'spdx': 'CECILL-C'},
     'ClarifiedArtistic': {'spdx': 'ClArtistic'},
+    'clearbsd': {'spdx': 'BSD-3-Clause-Clear'},
     'CommonPublicLicense10': {'spdx': 'CPL-1.0'},
+    'cpol': {'spdx': 'CPOL-1.02'},
     'Condor': {'spdx': 'Condor-1.1'},
     'ECL2.0': {'spdx': 'ECL-2.0'},
+    'eCos11': {'spdx': 'RHeCos-1.1'},
+    'eCos2.0': {'spdx': 'eCos-2.0'},
```

Comment:
This SPDX ID [is deprecated][1] in favor of [`GPL-2.0+ WITH eCos-exception-2.0`][2].  I'd rather use the modern license expression.

[1]: https://github.com/spdx/license-list-XML/blob/e5da40e25becb0aa7626d3f62649d2387284a623/src/eCos-2.0.xml#L7
[2]: https://github.com/spdx/license-list-XML/pull/392

Reply:
Do we want to allow license expressions for the 'spdx' tag or restrict it to the license ID's only?

Reply:
> Do we want to allow license expressions for the 'spdx' tag...

Yes, I think we do.  If someone finds `GPL-2.0+ WITH eCos-exception-2.0` content, they should be able to determine its FSF tags without going through a deprecated SPDX ID.

Reply:
OK - I'll add it to the amended PR - coming up soon

Reply:
I changed eCos2.0, but left ecos11 since RHeCos-1.1 is not marked as deprecated.